CHANGES.md

```diff
@@ -70,6 +70,14 @@ OpenSSL 3.5
 
    *David von Oheimb*
 
+ * `X509_PURPOSE_add()` has been fixed to take as the primary purpose identifier
+   not the `id` but the `sname` parameter.
```

Comment:
`` `X509_PURPOSE_add()` has been modified to take `sname` instead of `id` as the primary purpose identifier. ``